src/helpers/configuration_prompt_helpers.sh

```diff
@@ -22,34 +22,37 @@ function ensure_knows_configuration {
   fi
 
   local main_branch_input
+  local main_branch_current_value='None'
```

Comment:
I would lowercase this. `current value: none` reads better than `current value: None`, unless we write Visual Basic  :)